templates/infinispan-ephemeral.json

```diff
@@ -155,10 +239,25 @@
                         "fieldPath": "metadata.namespace"
                       }
                     }
+                  },
+                  {
+                    "name": "MGMT_USER",
+                    "value": "${MANAGEMENT_USER}"
+                  },
+                  {
+                    "name": "MGMT_PASS",
+                    "value": "${MANAGEMENT_PASSWORD}"
+                  },
+                  {
+                    "name": "APP_USER",
+                    "value": "${APPLICATION_USER}"
+                  },
+                  {
+                    "name": "APP_PASS",
+                    "value": "${APPLICATION_PASSWORD}"
                   }
                 ],
-                "image": "infinispan",
-                "imagePullPolicy": "Always",
+                "image": "${IMAGE}",
```

Comment:
why are you not doing this with an imagestream reference and imagechangetrigger anymore?